### PR TITLE
Update mozilla-firefox extension

### PR DESCRIPTION
**Pull request by Laptop765**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/mozilla-firefox/src/util/index.ts [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import fs from "fs";
 import path from "path";
+import { getPreferenceValues } from "@raycast/api";
+
+interface Preferences {
+  profileDirectorySuffix: string;
+}
```
Comment (pernielsentikaer) [inline extensions/mozilla-firefox/src/util/index.ts:7]: You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`x

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
This way could work @Laptop765, I'm not sure if there is a better way, but I guess it's hard to do anything better than this. @crisboarna, what do you think?

File: extensions/mozilla-firefox/src/util/index.ts (hunk 1 above)
Comment (Laptop765) [reply extensions/mozilla-firefox/src/util/index.ts]: Ah thanks!

**Comment by Laptop765** [comment]
> This way could work @Laptop765, I'm not sure if there is a better way, but I guess it's hard to do anything better than this. @crisboarna, what do you think?

We could parse `profiles.ini` as a separate fix but I'm not sure if that's better or not 😅

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**